engine/src/card.cpp

```diff
@@ -1088,30 +1088,43 @@ void MCCard::timer(MCNameRef mptr, MCParameter *params)
 		MCObject::timer(mptr, params);
 }
 
+bool MCCard::isdeletable()
+{
+    bool t_deletable = true;
+    
+    if (t_deletable)
+        t_deletable = MCObject::isdeletable();
+    
+    if (t_deletable)
+        t_deletable = !getflag(F_C_CANT_DELETE);
```

Comment:
Yes - you should be able to delete a stack regardless of the cantDelete status of its children. As you point out, deleting a stack is a little different from deleting an object on a stack - the former is removal from memory, the latter is obliteration.

To be fair, I think that there is a confusion in the current engine between 'user action commands' and 'script commands'. Why should script not be allowed to delete an object with cantDelete true? It makes no sense - script should be able to do whatever it wants.

The 'delete object' script command is trying to be both the 'user action' (which should be affected by cantDelete - which is more an editing property) and the general script command.